src/app/components/genres/genres.css

```diff
@@ -0,0 +1,9 @@
+.Genres {
```

Comment:
why it is in Uppercase and for example .footer  in lowercase. Css class namings should be a consistent system

Reply:
If you are using less in a project all components styles should be .less